### PR TITLE
Update package meta info

### PR DESCRIPTION
**Pull request by 304NotModified**

### CODE REVIEW

File: src/NLog.DiagnosticSource/NLog.DiagnosticSource.csproj [hunk 1]
```diff
@@ -14,19 +14,21 @@
 
     <Title>NLog.DiagnosticSource</Title>
     <Company>NLog</Company>
-    <Description>NLog Integration for Microsoft System.Diagnostics.DiagnosticSource and Activity </Description>
+    <Description>NLog Integration for Microsoft's System.Diagnostics.DiagnosticSource and Activity</Description>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/NLog/NLog.DiagnosticSource.git</RepositoryUrl>
     <PackageTags>DiagnosticSource;NLog;logging;log</PackageTags>
     <PackageIconUrl>https://nlog-project.org/N.png</PackageIconUrl>
     <PackageProjectUrl>https://github.com/NLog/NLog.DiagnosticSourcer</PackageProjectUrl>
```
Comment (snakefoot) [inline src/NLog.DiagnosticSource/NLog.DiagnosticSource.csproj:22]: Think also it should be https://github.com/NLog/NLog.DiagnosticSource without an "r" at the end

**Comment by snakefoot** [comment]
Have added net50 as targetframework and prepared for rc2 (Along with included changes from this PR)

But seems nuget-package fails to publish:

`Error publishing package. NuGet server returned 403: The specified API key is invalid, has expired, or does not have permission to access the specified package.`

**Comment by 304NotModified** [comment]
I need to update the key in the config, it's probably expired

**Comment by 304NotModified** [comment]
> But seems nuget-package fails to publish:
> 
> `Error publishing package. NuGet server returned 403: The specified API key is invalid, has expired, or does not have permission to access the specified package.`

Where do you see that error? anyway --> #22

**Comment by snakefoot** [comment]
Fixed by #11

**Pull request closed without merging**